TundraTD/Assets/Scripts/Level/LevelJudge.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Level
+{
+    /// <summary>
+    /// Handles the defeat and victory events and aftershocks
+    /// </summary>
+    public class LevelJudge : MonoBehaviour
+    {
+        [SerializeField] private Text resultOnEndScreen;
+        [SerializeField] private Button nextLevelButton;
+        [SerializeField] private Button retryButton;
+        [SerializeField] private Canvas endScreen;
+        [SerializeField] private PauseMode pauseMenu;
+        public void HandlePlayerDefeat()
+        {
+            Debug.Log("Player lost!");
+            retryButton.gameObject.SetActive(true);
+            resultOnEndScreen.text = "���������";
```

Comment:
Lol, just see how this text is displayed in GitHub
I guess we should implement localization in later builds to not make hard-coded values such as that